src/showplanner/Player.tsx

```diff
@@ -387,6 +383,11 @@ export function Player({
     if (id < PLAYER_COUNT) {
       return ("player-" + id) as LevelsSource;
     }
+    // We don't want the Preview channel doing anything silly, overwrite default player settings.
+    if (isPreviewChannel) {
+      dispatch(MixerState.setAutoAdvance({ player: id, enabled: false }));
+    }
```

Comment:
This doesn't seem right to me - if I'm reading it right `VUsource` is run as part of the render function, so this will get called every time. Surely it'd be better to ensure (in `MixerState`) that auto advance is never enabled for `PLAYER_ID_PREVIEW` in the first place?

Reply:
Possibly, but i'm not entirely sure how you change the default state depending on an invocation. Maybe I'm not looking hard enough.

Reply:
Oh, it shouldn't be in the VU source function regardless.

Reply:
Could you potentially ignore the request here if `action.payload.player === PLAYER_ID_PREVIEW`?

https://github.com/UniversityRadioYork/WebStudio/blob/87ca8a810626e009ffd68caee4cdb8085a49999a/src/mixer/state.ts#L290-L298

Reply:
Or maybe here would be better:

https://github.com/UniversityRadioYork/WebStudio/blob/87ca8a810626e009ffd68caee4cdb8085a49999a/src/mixer/state.ts#L863-L877

Reply:
Had a crack at a more flexible solution, seems to work.